db/seeds.rb

```diff
@@ -1,7 +1,34 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+articles = Article.create(
```

Comment:
Why are you wrapping it in a variable? Are you using it for anything?

Reply:
I thought it looked cleaner, to not repeat the code in every block.